### PR TITLE
Anna predev

### PR DESCRIPTION
**Pull request by amakarudze**
**Checklist**

- [x] My branch is up-to-date with the upstream `develop` branch.
- [x] I have added necessary documentation (if appropriate).

**Which issue does this PR fix?**: fixes #

Brief description of what this PR does.
Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber"

**Why do we need this PR?**:

If relevant, please include a screenshot.

**Demo (optional)**:

Some tips for you to write the instructions:
- Prefer bulleted description
- Start after checking out this branch
- Include any setup required, such as migrating databases, etc.

**Testing instructions:**
- Add SECRET_KEY to environment settings. 
- To test for different settings use "python manage.py runserver --settings=oshc.settings.filename" where filename can be dev, predev or production.

If there is any work still left to do, please add it here.

**TODOs (if any)**:
- I updated .gitignore with the .ve/ in the virtualenv section because that it what I had named my virtualenv and can be removed. 
- You would need to add SECRET_KEY to environment settings and there is need to add the instructions to the README.md of the project for contributors.

**A picture of a cute animal (not mandatory but encouraged)**:

### CODE REVIEW

**Comment by amakarudze** [comment]
os import is covered by "from .base import *".

File: oshc/oshc/settings/base.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        raise ImproperlyConfigured(error_msg)
+
+
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = get_env_variable('SECRET_KEY')
```
Comment (tapaswenipathak) [inline oshc/oshc/settings/base.py:32]: If you are removing the secret key, then you also need to enable it in for Travis. In ```.travis.yml```, you need to export like 
```
before_script:
  - export SECRET_KEY=<key>
```

Currently Travis is failing. You can keep the key as ```foobaroshc```.

**Review by tapaswenipathak — CHANGES_REQUESTED** [review]

**Comment by tapaswenipathak** [comment]
:wave: amakarudze, working on this?

**Comment by amakarudze** [comment]
Yes, I thought I had submitted a pull request for it some few weeks back. I
will check again.

Thanks.

**Pull request closed without merging**